dashboard-app/src/Error.tsx

```diff
@@ -67,7 +67,7 @@ const H1 = styled(_H1)`
   margin-bottom: ${SpacingEnum.medium};
 `;
 
-const H3Padded = styled(H3)`
+const H3 = styled(_H3)`
```

Comment:
This changes the spacing between elements. I've setup the `SpacingEnum` to follow the design principle of spaces being multiples of each other.

Reply:
I just changed the name of the variable, the spacing is the same as what you set it as.

Reply:
There are 2 H3 elements. Previously only one had padding attached to it, now both do.

Reply:
Oops, my bad. Will fix.